hyperdome_server/onion.py

```diff
@@ -509,71 +573,77 @@ def start_onion_service(self, port):
+            key_content = self.settings.get("private_key")
             key_type = "ED25519-V3"
         else:
             key_type = "NEW"
             key_content = "ED25519-V3"
 
-        debug_message = 'key_type={}'.format(key_type)
+        debug_message = "key_type={}".format(key_type)
         if key_type == "NEW":
-            debug_message += ', key_content={}'.format(key_content)
-        self.common.log('Onion', 'start_onion_service',
-                        '{}'.format(debug_message))
+            debug_message += ", key_content={}".format(key_content)
+        self.common.log("Onion", "start_onion_service", "{}".format(debug_message))
         await_publication = True
         try:
             res = self.c.create_ephemeral_hidden_service(
-                {80: port}, await_publication=await_publication,
-                key_type=key_type, key_content=key_content, timeout=10)
+                {80: port},
+                await_publication=await_publication,
+                key_type=key_type,
+                key_content=key_content,
+                timeout=10,
+            )
         except ProtocolError as e:
             raise TorErrorProtocolError(
-                strings._('error_tor_protocol_error').format(e.args[0]))
+                strings._("error_tor_protocol_error").format(e.args[0])
+            )
 
         self.service_id = res.service_id
-        onion_host = self.service_id + '.onion'
+        onion_host = self.service_id + ".onion"
 
         # A new private key was generated and is in the Control port response.
-        if self.settings.get('save_private_key'):
-            if not self.settings.get('private_key'):
-                self.settings.set('private_key', res.private_key)
+        if self.settings.get("save_private_key"):
+            if not self.settings.get("private_key"):
+                self.settings.set("private_key", res.private_key)
 
         if onion_host is not None:
             self.settings.save()
             return onion_host
         else:
-            raise TorErrorProtocolError(
-                strings._('error_tor_protocol_error_unknown'))
+            raise TorErrorProtocolError(strings._("error_tor_protocol_error_unknown"))
 
     def cleanup(self, stop_tor=True):
         """
         Stop onion services that were created earlier. If there's a tor
         subprocess running, kill it.
         """
-        self.common.log('Onion', 'cleanup')
+        self.common.log("Onion", "cleanup")
 
         # Cleanup the ephemeral onion services, if we have any
         if self.c:
             onions = self.c.list_ephemeral_hidden_services()
             for onion in onions:
                 try:
-                    self.common.log('Onion', 'cleanup',
-                                    'trying to remove onion {}'.format(onion))
+                    self.common.log(
+                        "Onion", "cleanup", "trying to remove onion {}".format(onion)
```

Comment:
slightly weird for this to be formatted differently than line 642

Reply:
black formats according to the set line length, if it fits in the line it fits, only if it doesn't fit will it split it up. The philosophy of using black is to not argue with it, so I'm not arguing with it even though it's a little silly.